app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -212,7 +209,15 @@ private fun ConversationScreenContent(
         messageText = messageText,
         onMessageChanged = onMessageChanged,
         onSendButtonClicked = onSendButtonClicked,
-        onSendAttachment = onSendAttachment,
+        onSendAttachment = {
+            coroutineScope.launch {
+                onSendAttachment(it)
+                // We add some extra delay to allowing some time to insert the message in the database, display it in the screen and be able
+                // to scroll to the bottom of the chat with the new list of messages
+                delay(MESSAGE_SCROLL_DELAY)
+                lazyListState.animateScrollToItem(0)
+            }
+        },
```

Comment:
looks a bit sketchy to me :P is there no other way to do it ? what if updating the database takes longer ? I know it shoudlnt be possible, but who knows. 

Why can't we control the scroll from the viewmodel ? as like send the attachment, wait some time for asynchronous sending and then once it is done scroll to the bottom ?